src/v2/Apps/Artwork/Components/ArtworkArtistSeries/index.tsx

```diff
@@ -62,14 +62,18 @@ export const ArtworkArtistSeriesFragmentContainer = createFragmentContainer<{
       internalID
       slug
       seriesArtist: artist(shallow: true) {
-        ...ArtistSeriesRail_artist
-        artistSeriesConnection {
+        # The below fragment is used for an exist-y check.
```

Comment:
The Relay compiler will rightly complain if this isn't so (the same field appearing w/ different args), but figured might as well add an explanatory comment.